### ISSUE TITLE
Release document 2.1

### ISSUE THREAD

**Issue by Styp** [pgRouting/pgrouting#434]
Nice to see that there is big development going on here!

Could you summarize all the 'new things' in 2.1?
Maybe even reference all the issue # ?

**Comment by cvvergara** [comment]
@Styp 
Here  are some interesting links:
The [2.1 documentation] (http://docs.pgrouting.org/2.1/doc/index.html)
The [2.1release notes] (http://docs.pgrouting.org/2.1/doc/src/changelog/2_1_0.html#changelog-2-1-0)
The [closed issues for 2.1] (https://github.com/pgRouting/pgrouting/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+milestone%3A%22Release+2.1.0%22+)
Some issues that were closed on 2.1 that we missed are [this ones](https://github.com/pgRouting/pgrouting/issues?utf8=%E2%9C%93&q=is%3Aissue+milestone%3A%22Release+2.2.0%22++label%3A%22Fixed+on+Develop%22+)
We send mails when releasing to the pgRouting users and developers list, you might want to subscribe to the the lists.
I'll leave this issue open for a week for people to read. Then I'll close it.

**Comment by cvvergara** [comment]
This issue is for V2.1, closing
